integration/airflow/openlineage/airflow/listener.py

```diff
@@ -0,0 +1,141 @@
+import logging
```

Comment:
Do we have unit tests for this `listener`?

Reply:
It's not a class 😉 

Also, it's hard to unit test this until we can run tests on released 2.3. 
Contrary to `ExtractorManager` it assumes things about Airflow and is not imported below 2.3.